trytouca/trytouca
# sdk-js: sub npm workspaces for lerna (#193)

QA:
- `npm run <cmd>` work as expected for all `scripts` in `/sdk/js/package.json`
- all tests in `/sdk/js/packages/node/test` passing
- `build` takes a bit longer since builds no longer run in parallel.  No effect on `build:prod` since examples don't build.
